### PR TITLE
GEODE-7716: geode-benchmarks src should be included in release artifacts

### PR DESCRIPTION
**Pull request by onichols-pivotal**
mailing list thread: https://lists.apache.org/thread.html/r67fb7bf3a9d249a0f9566a46bbd5667105e0ba564db3674a878d66d9%40%3Cdev.geode.apache.org%3E

DO NOT MERGE unless consensus has been reached and/or a vote has passed on the dev list.  Until then, this PR is provided only as an exhibit of the changes being proposed.

### CODE REVIEW

File: dev-tools/release/prepare_rc.sh [hunk 1]
```diff
@@ -228,10 +250,12 @@ mkdir ${FULL_VERSION}
 cp ${GEODE}/geode-assembly/build/distributions/* ${FULL_VERSION}
 cp ${GEODE_EXAMPLES}/build/distributions/* ${FULL_VERSION}
 cp ${GEODE_NATIVE}/build/apache-geode-native-${VERSION}* ${FULL_VERSION}
+cp ${GEODE_BENCHMARKS}/apache-geode-benchmarks-${VERSION}* ${FULL_VERSION}
+rm ${FULL_VERSION}/apache-geode-examples-*.zip*
```
Comment (upthewaterspout) [inline dev-tools/release/prepare_rc.sh]: Why this change to remove the examples zip?

Comment (onichols-pivotal) [reply dev-tools/release/prepare_rc.sh]: thanks, split this out into #4606

**Comment by onichols-pivotal** [comment]
@davebarnes97 if approved, perhaps the Apache Geode website should also have a link to this new artifact starting with 1.12.0
